packages/ui/components/auth.tsx

```diff
@@ -0,0 +1,95 @@
+import { AppConfig, UserSession, showConnect } from "@stacks/connect";
+// import { Storage } from '@stacks/storage';
+import { StacksMainnet, StacksTestnet } from "@stacks/network";
+import {
+  callReadOnlyFunction,
+  cvToJSON,
+  standardPrincipalCV,
+} from "@stacks/transactions";
+
+// Initialize Gaia hub permissions for the user
+const appConfig = new AppConfig(["store_write", "publish_data"]);
+
+// Set this to true if you want to use Mainnet
+// Using a boolean helps to switch the entire application to mainnet
+// or testnet with only changing a single value
+const isNetworkMainnet = true;
+
+export const userSession = new UserSession({ appConfig });
+// export const storage = new Storage({ userSession });
```

Comment:
did you mean to keep this comment?

Reply:
Yup, we will be needing the storage class to call any Gaia hub functions we may use in the future. So, I have added this export here for future use xD